src/utils.py

```diff
@@ -0,0 +1,125 @@
+import src.config as config
+import datetime as dt
+from dateutil import parser
+from pytz import timezone
+import os
+import pandas as pd
+
+
+def get_current_timestamp():
+    """Timestamp in milliseconds. Will be used by all scrapers in this or another form."""
+    return int(round(dt.datetime.now(dt.timezone.utc).timestamp() * 1000))
+
+
+def timestamp_to_datetime(timestamp):
+    return dt.datetime.utcfromtimestamp(int(timestamp / 1000))
+
+
+def timestamp_to_str(timestamp, format: ['date', 'exact_time']):
+    """
+    Converts timestamp into the desired format.
+    format = 'date' returns `2021-01-01` format.
+    format = 'exact_time' returns `2021-01-01 00:00:00` format.
+    """
+    if format == 'date':
+        return dt.datetime.utcfromtimestamp(int(timestamp / 1000)).strftime('%Y-%m-%d')
+    elif format == 'exact_time':
+        return dt.datetime.utcfromtimestamp(int(timestamp / 1000)).strftime('%Y-%m-%d %H:%M:%S+00:00') # UTC
+    else:
+        raise Exception('Format has to be either "date" or "exact_time".')
+
+
+def timestamp_utc_to_cet(datetime):
+    """
+    Twitter API returns datetime in UTC format by default. Use this function to convert UTC to CET time for consistency
+    between different scrapers. An alternative would be to use UTC everywhere.
```

Comment:
I would use UTC everywhere to avoid accidental errors, what do you think?

Reply:
Yep, actually already went for it